client/src/components/strat-overview/StratOverview.css

```diff
@@ -3,12 +3,14 @@
   flex-direction: row;
 }
 
-.preview-wrapper {
+.strat-info {
+  flex-basis:500px;
+  flex-grow: 2;
+}
+
+.strat-media {
   margin-right: 10px;
-  flex-direction: column;
-  align-content: center;
-  justify-content: center;
-  text-align: center;
-  display: flex;
-  min-width: 200px;
-}
\ No newline at end of file
+  min-width: 320px;
+  flex-basis:500px;
+  flex-grow: 4;
+}
```

Comment:
Not sure if this is doing what you want... the media items are getting stacked vertically so the info/videos are always just each taking up half the box horizontally afaict.
![image](https://github.com/celeste-library/celeste-speedrun-library/assets/30704914/047f0c24-19aa-448f-81ba-c1c86336d4f3)

Also small nit for consistent formatting, space after the colon on flex-basis